hearth/lib/hearth/http/error_parser.rb

```diff
@@ -91,7 +91,7 @@ def generic_error(error_opts)
         http_resp = error_opts[:http_resp]
         case http_resp.status
         when HTTP_3XX then @error_module::ApiRedirectError.new(
-          location: http_resp.headers['location'], **error_opts
+          location: http_resp.fields['location'], **error_opts
```

Comment:
This should still be `headers` right? (Or the `Field` needs to be converted to a string?)

Reply:
Good catch. Why didn't this fail in a test....